reddit.js

```diff
@@ -1,5 +1,37 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// Question 1
+var titles = reddit.data.children.forEach(function(titleName){
+	console.log(titleName.data.title);
+});
+// Question 2
+var links = reddit.data.children.map(function(link){
```

Comment:
Same case here as with above, child is probably a more appropriate parameter name than link.